multiqc/modules/merqury/merqury.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from MERQURY """
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+import re, base64
+from multiqc.plots import linegraph, table
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    """Merqury module"""
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(
+            name="MERQURY",
+            anchor="merqury",
+            href="https://github.com/marbl/merqury",
+            info="Evaluate genome assemblies with k-mers and more.",
+            doi="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02134-9",
+        )
+
+        # Find and load any MERQURY reports
+        self.completeness_data = dict()
+        for f in self.find_log_files("merqury/completeness"):
+            self.parse_completeness_log(f)
+
+        self.qv_data = dict()
+        for f in self.find_log_files("merqury/qv"):
+            self.parse_qv_log(f)
+
+        self.spectra_data = dict()
+        for f in self.find_log_files("merqury/spectra"):
+            self.parse_spectra_log(f)
+
+        log.info("Found {} completeness reports".format(len(self.completeness_data)))
+        log.info("Found {} qv reports".format(len(self.qv_data)))
+        log.info("Found {} spectra histograms".format(len(self.spectra_data)))
```

Comment:
I usually prefer to just log once - maybe with the maximum number of samples if you'd normally expect it to be the same value for all 3 types..?

Reply:
done